packages/back-end/src/services/features.ts

```diff
@@ -208,3 +209,24 @@ export function verifyDraftsAreEqual(
     );
   }
 }
+
+export async function ensureWatchingFeature(
```

Comment:
I made ensureWatching a shared function but I didn't restrain the type to only be "experiment" or "feature" in an effort to be more dynamic, since more things like metrics could be watched later.